cld2.go

```diff
@@ -23,3 +23,26 @@ func Detect(text string) string {
 	}
 	return lang
 }
+
+type LanguageDetection struct {
```

Comment:
`DetectedLanguage` would be a better name for this as it represents the result and not the action.